client-data/tools/pencil/pencil.js

```diff
@@ -149,6 +168,11 @@
 			"release": stopLineAt,
 		},
 		"draw": draw,
+		"stylus": false,
```

Comment:
This doesn't need to be globally available outside this tool if this is meant to be used only here, it can be a local variable.
If we want to be able to reuse this in other tools, then this should be `Tools.has_used_stylus` instead.

Reply:
Haven't thought about how this might work for other tools, so changed to a local variable for now.